signal/src/circular_buffer.h

```diff
@@ -0,0 +1,119 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef SIGNAL_SRC_CIRCULAR_BUFFER_H_
+#define SIGNAL_SRC_CIRCULAR_BUFFER_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+namespace tflm_signal {
+// TODO(b/286250473): remove namespace once de-duped libraries above
```

Comment:
We should probably still put this in a namespace. Does the naming conflict with the CircularBuffer op? We might want to consider renaming this.

Reply:
It doesn't conflict with CircularBuffer , I believe this has Audio specific stuff in it and isn't an op. It only clashes with the same file in AudioFrontend, would prefer not renaming anything till everything is ported over and we swap users to this package.

Reply:
I'm fine with avoiding renaming, but I think we should put this in the `tflite` namespace like everything else. Temporarily `tflite::tflm_signal` and eventually `tflite`.